awx/ui_next/src/screens/Inventory/InventoryList/InventoryListItem.jsx

```diff
@@ -85,108 +56,84 @@ function InventoryListItem({
-                <Badge isRead>{inventory.total_hosts}</Badge>
-              </ListGroup>
-              <ListGroup>
-                {i18n._(t`Sources`)}
-                <Badge isRead>{inventory.total_inventory_sources}</Badge>
-              </ListGroup>
-            </DataListCell>,
-            inventory.pending_deletion && (
-              <DataListCell alignRight isFilled={false} key="pending-delete">
-                <Label color="red">{i18n._(t`Pending delete`)}</Label>
-              </DataListCell>
-            ),
-          ]}
-        />
-        {!inventory.pending_deletion && (
-          <DataListAction
-            aria-label="actions"
-            aria-labelledby={labelId}
-            id={labelId}
-          >
-            {inventory.summary_fields.user_capabilities.edit ? (
-              <Tooltip content={i18n._(t`Edit Inventory`)} position="top">
-                <Button
-                  isDisabled={isDisabled}
-                  aria-label={i18n._(t`Edit Inventory`)}
-                  variant="plain"
-                  component={Link}
-                  to={`/inventories/${
-                    inventory.kind === 'smart' ? 'smart_inventory' : 'inventory'
-                  }/${inventory.id}/edit`}
-                >
-                  <PencilAltIcon />
-                </Button>
-              </Tooltip>
-            ) : (
-              ''
-            )}
-            {inventory.summary_fields.user_capabilities.copy && (
-              <CopyButton
-                copyItem={copyInventory}
-                isDisabled={isDisabled}
-                onCopyStart={handleCopyStart}
-                onCopyFinish={handleCopyFinish}
-                helperText={{
-                  tooltip: i18n._(t`Copy Inventory`),
-                  errorMessage: i18n._(t`Failed to copy inventory.`),
-                }}
-              />
-            )}
-          </DataListAction>
+    <Tr id={inventory.id} aria-labelledby={labelId}>
+      <Td
+        select={{
+          rowIndex,
+          isSelected,
+          onSelect,
+        }}
+        dataLabel={i18n._(t`Selected`)}
+      />
+      <Td id={labelId} dataLabel={i18n._(t`Name`)}>
+        {inventory.pending_deletion ? (
+          <b>{inventory.name}</b>
+        ) : (
+          <Link to={`${detailUrl}`}>
+            <b>{inventory.name}</b>
+          </Link>
         )}
-      </DataListItemRow>
-    </DataListItem>
+      </Td>
+      <Td dataLabel={i18n._(t`Status`)}>
+        {inventory.kind !== 'smart' && <StatusLabel status={syncStatus} />}
+      </Td>
+      <Td dataLabel={i18n._(t`Type`)}>
+        {inventory.kind === 'smart'
+          ? i18n._(t`Smart Inventory`)
+          : i18n._(t`Inventory`)}
+      </Td>
+      <Td key="organization" dataLabel={i18n._(t`Organization`)}>
+        <Link
+          to={`/organizations/${inventory.summary_fields.organization.id}/details`}
+        >
+          {inventory.summary_fields.organization.name}
+        </Link>
```

Comment:
```suggestion
        <Link
          to={`/organizations/${inventory?.summary_fields?.organization?.id}/details`}
        >
          {inventory?.summary_fields?.organization?.name}
        </Link>
```

@keithjgrant We'll need to handle the intermediate state for list items when they're being deleted. It appears that inventories can lose their reference to the related organization during that time:

![Screenshot from 2020-12-17 13-04-42](https://user-images.githubusercontent.com/9753817/102526456-ab066980-4069-11eb-8104-97ce1fbcb785.png)

(Found w/ the latest e2e/visual run :tada: ! :tada: !)